Google-Summer-of-Code-2023.md

```diff
@@ -273,3 +273,504 @@ The following is a list of Oppia's 2023 GSoC project ideas. You are welcome to c
+
+**Dependency on Release Schedule:** Some sections of this proposal may entail writing Beam jobs to update existing server data. The timeline should be arranged so that such jobs can be run and verified during the appropriate release cycle.
+
+**Proposal notes:**
+
+* The main thing that is important to demonstrate in the proposal for this project is good technical design skills. Strong proposals would first show a good understanding of the current system, and correctly describe the parts of it that are relevant to the relevant subproject, before suggesting the minimal changes that would be needed in order to achieve the desired functionality.
+* For (b), some precomputation may be needed in order to retrieve the version of the "previous change" quickly.
+* For (b), it would be useful to generalize the system so that one can go forward/back from any given state. This would allow additional useful functionality like clicking on a state in a particular version when it's shown in the history tab, and moving forward/back through its history. Be sure to handle state additions, deletions and renames correctly!
+
+**Useful resources:**
+
+* How to write Apache Beam jobs: [wiki page](https://github.com/oppia/oppia/wiki/Apache-Beam-Jobs)
+* Release schedule: [wiki page](https://github.com/oppia/oppia/wiki/Release-schedule-and-other-information)
+
+### 1.3. Serial chapter launch
+
+TBA
+
+### 1.4. User feedback reporting UI on Web
+
+TBA
+
+### 1.5. Learner dashboard redesign
+
+TBA
+
+### 1.6. Improve math interactions
+
+TBA
+
+### 1.7. Accessibility improvements
+
+TBA
+
+## Contributor Dashboard team
+
+### 2.1. Contributor Dashboard Admin Stats Table
+
+**Project Description:**
+
+Contributor dashboard admin page revamp. Table view of contributors with key stats, e.g. number of translations/questions submitted/reviewed. Each row represents a contributor and is actionable, allowing an admin to add or remove contribution rights. 
+
+**Size of this project:** medium (~175 hours)
+
+**Potential Mentors:** 
+
+**Knowledge/Skills Recommended:**
+
+* Knowledge and understanding of Python
+* Knowledge and understanding of TypeScript, Angular, basic HTML, and CSS
+
+**Suggested Milestones:**
+
+* **Milestone 1:** Complete the complete backend changes:
+  - Introduce language coordinator admin role
+  - Modify contributor dashboard admin controller to allow adding or removing language coordinators
+  - Add new endpoint to contributor dashboard admin controller for fetching contributor stats
+
+* **Milestone 2:** Complete the frontend changes:
+  - Display contributor stats table on contributor dashboard admin page
+  - Show different available admin actions depending on user admin role
+  - Add support for sorting by table column (will require some backend query changes)
+  - Implement table pagination
+
+
+**Dependency on Release Schedule:** 
```

Comment:
We consider a milestone to be done when satisfactory changes are merged into develop, right? (i.e. they don't need to be deployed to production) If so, can we put "None" here?

ditto for the other blank sections. I think it's good to put "None" to make clear that we didn't forget about it